File: Examples/CSharp/ProgrammersGuide/Load-Save-Convert/ExportToXAML.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CSharp.ProgrammersGuide.Load_Save_Convert
{
    public class ExportToXAML
    {
        public static void Run()
        {
            // ExStart:ExportToXAML
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadSaveConvert();
            // load diagram
            Diagram diagram = new Diagram(dataDir + "ExportToXAML.vsd");
            // save diagram
            diagram.Save(dataDir + "Output.xaml", SaveFileFormat.XAML);
            // ExEnd:ExportToXAML
        }
    }
}

File: Examples/VisualBasic/Load-Save-Convert/ExportToXAML.vb

Imports Aspose.Diagram
Imports System

Public Class ExportToXAML
    Public Shared Sub Run()
        ' ExStart:ExportToXAML
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_LoadSaveConvert()
        ' load diagram
        Dim diagram As New Diagram(dataDir & Convert.ToString("ExportToXAML.vsd"))
        ' save diagram in the XAML format
        diagram.Save(dataDir & Convert.ToString("Output.xaml"), SaveFileFormat.XAML)
        ' ExEnd:ExportToXAML
    End Sub
End Class
